src/ranking.service.ts

```diff
@@ -0,0 +1,21 @@
+import{AxiosInstance} from "axios";
+import {Keywords, InputArchive} from "./types/ranking";
+
+const endpoint = "locations";
+
+export default (axios: AxiosInstance) => {
+    return{
+        addKeywordsLocation: (params: Keywords) => {
+            return axios.post(endpoint + "/keywords", params);
+        },
+        listKeywords: (locationId: Keywords) => {
+            return axios.get(endpoint + "/" + locationId + "/keywords");
+        },
+        getKeywordsPerformance: (locationId: Keywords) => {
```

Comment:
param type should be KeywordsPerformance and param name should be more descriptive